docs/Migrating-v3.md

```diff
@@ -0,0 +1,101 @@
+# ECL v3 migration notes
+
+The following guidelines aim to facilitate migration between ECL v2 to v3.
+
+## Deprecated components haven't been migrated
+
+The following have been kept in v2 in order to avoid breaking changes, but will no longer be available in v3:
+
+Components:
+
+- Accordion (deprecated since ECL 2.6.0)
+- Footer (deprecated since ECL 2.12.0)
+- Menu (legacy)
+- Page header (deprecated since ECL 2.14.0)
+- Site header (deprecated since ECL 2.12.0)
+- Timeline (deprecated since ECL 2.5.0)
+
+Templates:
+
+- Content page (deprecated sine ECL 2.12.0)
+- Standard page (deprecated sine ECL 2.12.0)
+
+## npm packages
+
+All packages are under [`@ecl`](https://www.npmjs.com/org/ecl) organization. System-specific prefixes have been removed.
+
+- SCSS packages have the following change in pattern: `@ecl/(ec|eu)-component-{component_name}` has become `@ecl/vanilla-component-{component_name}`.
+
+Example: `@ecl/ec-component-accordion` => `@ecl/vanilla-component-accordion`
+
+- Twig templates packages have the following change in pattern: `@ecl-twig/(ec|eu)-component-{component_name}` has become `@ecl/twig-component-{component_name}`.
+
+Example: `@ecl-twig/ec-component-accordion` => `@ecl/twig-component-accordion`
+
+- Resources' packages have the following change in pattern: `@ecl/(ec|eu)-resources-{resource_name}` has become `@ecl/resources-(ec|eu)-{resource_name}`
+
+Example: `@ecl/ec-resources-logo` => `@ecl/resources-ec-logo`
+
```

Comment:
should we also mention the renaming of specs packages?
`@ecl/ec-specs-accordion` => `@ecl/specs-component-accordion`

Reply:
I have deliberately left specs out of this document. Specs are used only internally and adding information about them will only add noise. I doubt we'll even publish them on npm now having twig templates in the repo.